sendfile/backends/_internalredirect.py

```diff
@@ -19,6 +19,6 @@ def _convert_file_to_url(filename):
         url.insert(1, head)
 
     # Python3 urllib.parse.quote accepts both unicode and bytes, while Python2 urllib.quote only accepts bytes.
-    # So force bytes for quoting and then go back to unicode.
-    url = [force_bytes(url_component) for url_component in url]
-    return force_text(quote(b'/'.join(url)))
+    # So use bytes for quoting and then go back to unicode.
+    url = [smart_bytes(url_component) for url_component in url]
+    return smart_text(quote(b'/'.join(url)))
```

Comment:
The only minor thing here is that the byte literal does tie this to Python 2.7+.  I guess that's not necessarily a bad thing and makes supporting Python 3 easier I guess.